template/_prettierrc.js

```diff
@@ -1,7 +1,7 @@
 module.exports = {
+  arrowParens: 'avoid',
+  bracketSameLine: true,
   bracketSpacing: false,
-  jsxBracketSameLine: true,
```

Comment:
I know that this has been deprecated, but since this template aims to be consistent with the default RN template, I advise you to create a PR for that in the RN repo: https://github.com/facebook/react-native/blob/v0.66.4/template/_prettierrc.js

Reply:
Makes sense! I only checked that `main` branch in `react-native`, which has this addressed already.